### DISCUSSION TITLE
Sprint Demo Recordings

### DISCUSSION THREAD

**Discussion by tundeojediran** [t1-loanwise/loan-wise-Group1-Team1A#23] [General]
@t1-loanwise/a-team @t1-loanwise/b-team 
Here is a link to today's Sprint Demo recording.
https://tldv.io/app/meetings/647077337afee70012b3f4b5

**Comment by moni-nancy** [comment]
Thank you sir

On Fri, May 26, 2023 at 12:17 PM Daniel Tunde Ojediran <
***@***.***> wrote:

> @t1-loanwise/a-team <https://github.com/orgs/t1-loanwise/teams/a-team>
> @t1-loanwise/b-team <https://github.com/orgs/t1-loanwise/teams/b-team>
> Here is a link to today's Sprint Demo recording.
> https://tldv.io/app/meetings/647077337afee70012b3f4b5
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/t1-loanwise/discussions/23>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWTZUVYBADDFZCXV3C7GWJLXICGNTANCNFSM6AAAAAAYQC4VUQ>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID: ***@***.***>
>

-- 
All communications with this email account are bound by the policies and 
laws of the Federal University of Agriculture, Abeokuta. Kindly go to the 
http://www.unaab.edu.ng/policy 
<http://www.unaab.edu.ng/attachments/Funnabictpolicy.pdf> to view details

**Comment by geliettech** [comment]
Thank you sir
